source/fuzz/fuzzer_pass_add_bit_instruction_synonyms.cpp

```diff
@@ -64,6 +64,53 @@ void FuzzerPassAddBitInstructionSynonyms::Apply() {
           continue;
         }
 
+        auto ir_context = GetIRContext();
```

Comment:
I agree with @paulthomson that it'd be better to refactor duplicated code into a separate method. Usually, when we have duplications like this, we create a public static method in the transformation class and put all the duplicated code there. Then, we can call that method in `IsApplicable` as well as `FuzzerPass::Apply`. You can check out an example in `TransformationAddCopyMemory`.

Reply:
OK, I will do that.

Reply:
Done.